function.k

```diff
@@ -359,10 +404,51 @@ rule <k> pcsContractPart(C:Id, function F:Id (P1:FParameters) .FunQuantifiers
 	       ...
            </Function>
 
+      rule <k> UpdateTemp(CI) => . ... </k>   
+           <contractStack> ListItem(CI:Int) ... </contractStack>
+           <contractInstance>
+     	       <ctId> CI </ctId>
+               <ctContext> Rho </ctContext>
+               <ctstore> S </ctstore>
+               <distore> Ds </distore>
+               <Memory> M </Memory>
+               <tempContext> Rho1=>Rho </tempContext>
+               <tempStore> S1=>S </tempStore>
+               <tempdistore> Ds1 => Ds </tempdistore>
+               <tempmemory> M1 => M </tempmemory>
+               ...
+           </contractInstance>  
+           <funCond> _=>true </funCond>
+           <forInit> _=>true </forInit>
 
-     
-    
-      rule <k> functionCall(Sender:Int; C:Int ; F:Id ; Es:Values ; M:Msg); => FunCall(C:Int, F:Id, Es:Values,M:Msg) ~> . ... </k>
+      rule <k> throw; => . ... </k>
+           <contractStack> ListItem(CI:Int) ... </contractStack>
+           <contractInstance>
+     	       <ctId> CI </ctId>
+               <ctContext> Rho => Rho1 </ctContext>
+               <ctstore> S => S1 </ctstore>
+               <distore> Ds => Ds1 </distore>
```

Comment:
Again, I think we should improve the naming. 
What is `distore` doing? Not clear from the name.